tests/sb-util.test.ts

```diff
@@ -134,5 +141,44 @@ describe('Sprite class -------------------------', () => {
 	test('can get lists', () => {
 		const lists = sprite.lists();
 		expect(lists).toBeInstanceOf(Object);
-	})
+	});
+
+	test('can get all blocks', () => {
+		const blocks = sprite.blocks();
+		expect(blocks).toBeInstanceOf(BlockCollection);
+	});
+});
+
+describe('BlockCollection class', () => {
+	let sp, blocks;
+
+	beforeAll(async () => {
+		// Choosing this file because it has a lot of blocks
+		sp = await loadProjectJson(`${process.cwd()}/tests/data/accelerator.json`);
+		blocks = sp.blocks();
+	});
+
+	test('query for opcode', () => {
+		const eventFlagBlocks = blocks.query('event_whenflagclicked');
+		expect(eventFlagBlocks).toBeInstanceOf(BlockCollection);
+		expect(eventFlagBlocks.first()).toBeInstanceOf(Block);
+	});
+
+	test('query for invalid opcode', () => {
```

Comment:
Does `invalid` mean not present? I'm not sure if there is an invalid opcode or how you might tell.